src/taco-dependency-installer/test/androidSdkInstaller.ts

```diff
@@ -0,0 +1,124 @@
+/**
+ *******************************************************
+ *                                                     *
+ *   Copyright (C) Microsoft. All rights reserved.     *
+ *                                                     *
+ *******************************************************
+ */
+
+/// <reference path="../../typings/dependencyInstallerInterfaces.d.ts"/>
+/// <reference path="../../typings/mocha.d.ts"/>
+/// <reference path="../../typings/mockery.d.ts"/>
+/// <reference path="../../typings/should.d.ts"/>
+
+"use strict";
+
+/* tslint:disable:no-var-requires */
+// var require needed for should module to work correctly
+// Note not import: We don't want to refer to shouldModule, but we need the require to occur since it modifies the prototype of Object.
+var shouldModule = require("should");
+/* tslint:enable:no-var-requires */
+
+import installerProtocol = require("../elevatedInstallerProtocol");
+import ILogger = installerProtocol.ILogger;
+import mockery = require("mockery");
+import nodeFakes = require("../../taco-tests-utils/nodeFakes");
```

Comment:
You shouldn't have paths that escape the package. If you want to reference this file, it should be via a dev/optional dependency on the other package, I think.